Test/DependencyCollector/FunctionalTests/TestApps/Aspx451/Aspx451.csproj

```diff
@@ -104,7 +106,7 @@
       <HintPath>..\..\..\..\..\..\packages\Microsoft.AspNet.ScriptManager.WebForms.5.0.0\lib\net45\Microsoft.ScriptManager.WebForms.dll</HintPath>
     </Reference>
     <Reference Include="System.Web.Optimization, Version=1.1.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35">
-      <HintPath>..\..\..\..\..\..\packages\Microsoft.AspNet.Web.Optimization.1.1.3\lib\net45\System.Web.Optimization.dll</HintPath>
+      <HintPath>..\..\..\..\..\..\packages\Microsoft.AspNet.Web.Optimization.1.1.3\lib\net40\System.Web.Optimization.dll</HintPath>
```

Comment:
why net40?

Reply:
Same as above - reverted the previous mistake